numba/tests/test_np_functions.py

```diff
@@ -3906,7 +3906,11 @@ def test_asarray_chkfinite(self):
             # 3x2 (with higher order broadcasting)
             (
                 np.arange(36).reshape(6, 2, 3),
-            )
+            ),
+            # (
```

Comment:
I forgot to remove this comments here and made another commit after this to remove this

Reply:
Yeah, I think that commit was lost, I made another one to fix it, thanks!